x/nodes/keeper/reward.go

```diff
@@ -4,147 +4,48 @@ import (
 	"fmt"
 	"github.com/pokt-network/pocket-core/x/nodes/types"
 	sdk "github.com/pokt-network/posmint/types"
-	"github.com/pokt-network/posmint/x/auth"
 	govTypes "github.com/pokt-network/posmint/x/gov/types"
-	"github.com/tendermint/go-amino"
 )
 
 // award coins to an address (will be called at the beginning of the next block)
 func (k Keeper) RewardForRelays(ctx sdk.Ctx, relays sdk.Int, address sdk.Address) {
-	award, _ := k.getValidatorAward(ctx, address)
 	coins := k.RelaysToTokensMultiplier(ctx).Mul(relays)
-	k.setValidatorAward(ctx, award.Add(coins), address)
-	ctx.Logger().Info("Custom award of " + coins.String() + " set for " + address.String())
+	toNode, toFeeCollector := k.NodeCutOfReward(ctx, coins)
+	k.mint(ctx, toNode, address)
+	k.mint(ctx, toFeeCollector, k.getFeePool(ctx).GetAddress())
```

Comment:
Check if feeCollector values are 0 as they are params and could be zero'd out by governance.

Reply:
Well, should work either way, but will add for sanity